### PR TITLE
Base Classes added

### LINKED ISSUE #35: Add base Data Classes
Add classes for Recipe, Ingredient, and MealPlan

### PR DESCRIPTION
**Pull request by jeastgaard**
MealPlan and Recipe classes have been added with minimal implimentation.

### CODE REVIEW

File: app/src/main/java/com/example/happymeals/FireStoreManager.java [hunk 1]
```diff
@@ -29,6 +29,12 @@
  */
 public class FireStoreManager {
 
+    // <todo> Move enumerations over to a constants class.
+    public enum COLLECTION_NAME {
```
Comment (kstark1) [inline app/src/main/java/com/example/happymeals/FireStoreManager.java]: Move enums into constant class

File: app/src/main/java/com/example/happymeals/Recipe.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+ * @author jeastgaa
+ * @version 1.00.01
+ * @see DatabaseObject The parent object for this class which allows it to be stored inside the
+ * firebase databse using the {@link FireStoreManager}
+ * This class represents recipes that can be loaded inside the application and shown to the
+ * end user.
+ */
+public class Recipe extends DatabaseObject {
+
+    private double cookTime;
```
Comment (kstark1) [inline app/src/main/java/com/example/happymeals/Recipe.java:19]: Need comments attribute see user story [US 02.01.01 Add ability to add recipes#9](https://github.com/CMPUT301F22T32/Happy-Meals/issues/9)

File: app/src/main/java/com/example/happymeals/MealPlan.java [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+ * firebase databse using the {@link FireStoreManager}
+ * This class represents Meal Plans that can be loaded inside the application and shown to the
+ * end user. These meal plans will always be stored in the database as a week/week schema. This
+ * means that the meal plan should be defined with the date as a Sunday value otherwise the
+ * data will not turn out.
+ */
+public class MealPlan extends DatabaseObject {
+
+    //<todo> Move enumeration over to a constants class.
+    public enum DAY_OF_WEEK {
```
Comment (kstark1) [inline app/src/main/java/com/example/happymeals/MealPlan.java:29]: Add to constant class

File: app/src/main/java/com/example/happymeals/MealPlan.java [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+        SUNDAY,
+        MONDAY,
+        TUESDAY,
+        WEDNESDAY,
+        THURSDAY,
+        FRIDAY,
+        SATURDAY
+    };
+
+    public enum MEAL_OF_DAY {
```
Comment (kstark1) [inline app/src/main/java/com/example/happymeals/MealPlan.java:39]: add to constant class

**Review by kstark1 — APPROVED** [review]

File: app/src/main/java/com/example/happymeals/FireStoreManager.java (hunk 1 above)
Comment (jeastgaard) [reply app/src/main/java/com/example/happymeals/FireStoreManager.java]: Will do in integration ticket.

**Pull request merged**